tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2203,6 +2186,141 @@ public void handlePath(String xpath) {
+                if (vxmlFile == null) {
+                    return; // use baseline
+                }
+                String vxmlValue = vxmlFile.getStringValue(xpath);
+                if (vxmlValue == null) {
+                    throw new RuntimeException("vxmlValue == null");
+                }
+                // steps >= 2 identifies "L2+" locales
+                boolean revertToBaseline = /* (steps >= 2) && */ wantRevertToBaseline(xpath, vxmlValue);
+                if (!revertToBaseline) {
+                    String fullXPath = vxmlFile.getFullXPath(xpath);
+                    replace(fullXPath, fullXPath, vxmlValue);
+                }
+            }
+
+            /**
+             * 	Revert each L2+ locale/path VXML value of ↑↑↑ to the baseline value, where all the following criteria are met:
+             * 	1	The baseline value is a hard value (can use unresolved file for this)
+             * 	2	The survey tool did not allow changes in the locale/path in v43
+             * 	    ▪ That is, other than allowed locales with Person Names, Turkey, missing, and errors
+             *    	▪ NOTE: we should be able to check missing/errors by running the CheckCLDR tests on the baseline data. Most locales only had PN & Turkey, but some had errors or missing values.
+             * 	3	The bailey value is fetched from a different path AND from an ancestor locale.
+             * 	    ▪ Ideally we would look at intervening paths also, but I think this is sufficient.
+             */
+            private boolean wantRevertToBaseline(String xpath, String vxmlValue) {
+                if (!CldrUtility.INHERITANCE_MARKER.equals(vxmlValue)) {
+                    // criterion zero: if vxml value is not ↑↑↑, don't revert to baseline
+                    return false;
+                }
+                // String baselineValue = baselineFileResolved.getStringValue(xpath);
+                String baselineValue = baselineFileUnresolved.getStringValue(xpath);
+                if (baselineValue == null || CldrUtility.INHERITANCE_MARKER.equals(baselineValue)) {
+                    // criterion 1: if baseline value is not a hard value, don't revert to baseline
+                    return false;
+                }
+                String localeID = cldrFileToFilter.getLocaleID();
+                String fullXPath = vxmlFile.getFullXPath(xpath);
+                if (changesWereAllowed(localeID, xpath, fullXPath)) {
+                    // criterion 2: if Survey Tool did allow changes in the locale/path in v43, don't revert to baseline
+                    return false;
+                }
+                Output<String> inheritancePathWhereFound = new Output<>();
+                Output<String> localeWhereFound = new Output<>();
+                baselineFileResolved.getBaileyValue(xpath, inheritancePathWhereFound, localeWhereFound);
+                if (localeID.equals(inheritancePathWhereFound.value) || xpath.equals(inheritancePathWhereFound.value)) {
```

Comment:
IMPORTANT! Just discovered bug here: 
localeID.equals(inheritancePathWhereFound.value)
should be
localeID.equals(inheritanceLocaleWhereFound.value)

Reply:
I'll fix that...